deploy/README.md

```diff
@@ -68,9 +77,146 @@ The diagram below illustrates the components of the Kubernetes collection soluti
+```
+
+For example, if your release name is `my-release` and namespace is `my-namespace`:
+```bash
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/sumologic/values.yaml | \
+sed 's/\-sumologic.sumologic'"/-sumologic.my-namespace/g" | \
+sed 's/\collection'"/my-release/g" > values.yaml
+```
+
+Make any changes to the `values.yaml` file as needed, and run the following to install the chart with the override file.
+
+```bash
+helm install sumologic/sumologic --name my-release --namespace my-namespace -f values.yaml
+```
+### How to install if you have an existing Prometheus operator
+
+Run the following to download the `values.yaml` file
+
+```bash
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/master/deploy/helm/sumologic/values.yaml
+```
+
+Edit the `values.yaml` file so `prometheus-operator.enabled = false`, and run
+
+```bash
+helm install sumologic/sumologic --name collection --namespace sumologic -f values.yaml
+```
+
+If you already have a customized remote write configuration you’ll need to make some manual changes, see the [manual configuration](#manual-configuration-steps) steps, otherwise follow the [default](#default-steps) steps.
+
+#### Default steps
+
+Run the following to update the [remote write configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) of the prometheus operator by installing with the prometheus overrides file we provide.
+
+```bash
+curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v0.4.0/deploy/helm/prometheus-overrides.yaml
+```
+
+Then run
+
+```bash
+helm upgrade prometheus-operator stable/prometheus-operator -f prometheus-overrides.yaml
+```
+
+#### Manual configuration steps
+
+Helm supports providing multiple configuration files, priority will be given to the last (right-most) file specified. You can obtain your current prometheus configuration by running
+
+```bash
+helm get values prometheus-operator > current-values.yaml
+```
+
+Any section of `current-values.yaml` that conflicts with sections of our `prometheus-overrides.yaml` will have to be removed from the `prometheus-overrides.yaml` file and appended to `current-values.yaml` in relevant sections. For any config that doesn’t conflict, you can leave them in `prometheus-overrides.yaml`. Then run
+
+```bash
+helm upgrade prometheus-operator stable/prometheus-operator -f current-values.yaml -f prometheus-overrides.yaml
+```
+
+### How to install if you have standalone Prometheus
+
+Update your Prometheus configuration file’s `remote_write` section, as per the documentation here: [https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) by taking the `remoteWrite` section of the `prometheus-overrides.yaml` file, and making the following changes:
```

Comment:
nit: can we do `[here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write)` instead of the link?